src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowSkeleton.java

```diff
@@ -107,10 +107,6 @@ public void onButtonClicked(@NotNull final Button button)
             buttons.get(button.getID()).accept(button);
             Network.getNetwork().sendToServer(new ClickGuiButtonTriggerMessage(button.getID(), this.resource));
         }
-        else
```

Comment:
Why was this removed?!?!?!??!!??!?!?!?!

Reply:
Because we do not handle all buttons like this and it spams the console when we don't.